src/main/resources/application.properties

```diff
@@ -58,46 +52,44 @@ spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.mongo.embedd
 # Check configuration documentation for using jasypt encrypted passswords
 # Uri examples with encryption:
 # Example 1: 
-# spring.data.mongodb.password: ENC(en2h9ZQlZEuWyyfiTVrOPcQbRTwcXxOJ)                               - The encrypte password property
+# spring.data.mongodb.password: ENC(en2h9ZQlZEuWyyfiTVrOPcQbRTwcXxOJ)                               - The encrypted password property
 # spring.data.mongodb.uri: mongodb://username:${spring.data.mongodb.password}@localhost:27017       - Single MongoDD instance with tag for password property
 
 spring.data.mongodb.uri: mongodb://localhost:27017
 spring.data.mongodb.database: eiffel_intelligence
-
-# session expiration
 # Configure the session storage expiration time for logged in users of Eiffel Intelligence front-end
-server.session-timeout: 1200
+server.session.timeout: 1200
 # Configure the database collection name to store sessions
 sessions.collection.name: sessions
 
 # Name of the database collection where aggregated objects are stored
-aggregated.collection.name: aggregated_objects
+aggregations.collection.name: aggregations
 # Configure time-to-live value (seconds) for aggregated objects. This needs to be set if
 # using the example rule set 'AllEventsRules' to avoid the storage growing too large.
 # Recommended setting is 10 minutes.
-aggregated.collection.ttlValue:
+aggregations.collection.ttl:
 # The name of the database collection where to store the mapping between
 # aggregated objects and id of the Eiffel events that contributed to those
-event_object_map.collection.name: event_object_map
+event.object.map.collection.name: event_object_map
 # Name of the database collection where subscriptions are stored
```

Comment:
This file has now become quite large. I'm partly to blame since I extended some comments in my last PR but I think we need to reduce some of these comments and instead refer to configuration.md. The documentation can instead incude more explicit examples of how some properties.

Reply:
Should I remove all comments and just keep documentation in configuration.md?

Reply:
A lot of comments can be removed I think, since they are self explanatory ... but perhaps those with warnings attached to them should be kept?

Reply:
I removed most of the comments and added some extra text in configurations.md